src/Routes/RouteConfigurator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace  Kanvas\Routes;
+
+use Kanvas\Routes\PublicRoutes;
+use Kanvas\Routes\PrivateRoutes;
+
+/**
+ * Route Configurator Class
+ */
+class RouteConfigurator
```

Comment:
@rwhite27 by using the function we dont have to create a provider just specify it on the documentation